src/content/en/2020/caching.md

```diff
@@ -4,11 +4,12 @@ part_number: IV
 chapter_number: 20
 title: Caching
 description: Caching chapter of the 2020 Web Almanac covering cache-control, expires, TTLs, validitaty, vary, set-cookies, AppCache, Service Workers and opportunities.
-authors: [roryhewitt]
-reviewers: [raghuramakrishnan71, csswizardry, jzyang, jaisanth, Soham-S-Sarkar]
+authors: [roryhewitt, raghuramakrishnan71]
+reviewers: [csswizardry, jzyang, jaisanth, Soham-S-Sarkar]
 analysts: [raghuramakrishnan71]
 translators: []
 roryhewitt_bio: Enterprise Architect at <a href="https://www.akamai.com/">Akamai</a>. Passionate about performance.
+#raghuramakrishnan71_bio: TODO
```

Comment:
@raghuramakrishnan71 I was chatting to @roryhewitt and he'd like to have you as co-author given all the amazing work you did to get this Chapter over the finish line.

Could you suggest a short bio for yourself? You can use HTML (not markdown I'm afraid).

Reply:
@raghuramakrishnan71 I'm going to go ahead and merge this in case you don't see this in time. Could you open a separate PR for your bio?